misc/dbt-materialize/dbt/include/materialize/macros/mz_create_index.sql

```diff
@@ -0,0 +1,49 @@
+-- Copyright Materialize, Inc. and contributors. All rights reserved.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License in the LICENSE file at the
+-- root of this repository, or online at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+{% macro mz_create_index(obj_name, default=False, idx_name="", col_refs=[], with_options=[]) -%}
+  -- todo@jldlaughlin
+  -- figure out docs and hooks!
```

Comment:
not sure if this matters, and it definitely makes more sense to do whatever is common, but you ought to be able to use jinja comments like `{# todo@jldlaughlin #}` and the todos won't get put into the generated code.

Not sure if the generated code is ever visible, if it's not then it definitely doesn't matter.

Reply:
the generated code will show up in logs -- thanks for the suggestion!